src/components/KonnektSearch.vue

```diff
@@ -5,20 +5,20 @@
         v-layout(row fill-height)
           v-flex(lg6 offset-xs3)
             v-text-field(
-              v-model="search"
+              v-model="search_1"
               outline
               prepend-inner-icon="mdi-magnify"
               label="search any skill or name"
               clearable)
           v-flex
-            v-btn(large color="primary") GO
-    v-container
-      div(v-for="person in filteredList" v-bind:key="person.name")
+            v-btn(large color="primary" to="") GO
+    v-container(v-if="!$apollo.loading")
+      div(v-for="(node,i) in search.edges" :key="i")
```

Comment:
Check if `({ node }, i) in search.edges` work, you'll not have to do `node.node` in many places then.

Reply:
works nicely. changing to {node} instead of node.

Reply:
done :)